docs/checks.md

```diff
@@ -6,194 +6,201 @@ This is a list of vulnerability checks that can be used in the configuration for
 
 > **Note!** Each of the commands here can check for the opposite by appending 'Not' to the check type. For example, `PathExistsNot` to pass if a file does not exist.
 
-**Command**: pass if command succeeds (exit code `0`, checks `$?`)
+> **Note!** If a check has negative points assigned to it, it automatically becomes a penalty.
 
-```
-type='Command'
-arg1='grep "pam_history.so" /etc/pam.d/common-password'
-```
+> **Note!** Each of these check types can be used for `Pass`, `PassOverride` or `Fail` conditions, and there can be multiple conditions per check. See [configuration](config.md) for more details.
 
-> **Note!** Each of these check types can be used for either `Pass` or `Fail` conditions, and there can be multiple conditions per check.
 
-**CommandOutput**: pass if command output matches exactly. if error, never passes
+> Note: `Command*` checks are prone to interception, modification, and tomfoolery. Your scoring configuration will be much more robust if you rely on checks using native mechanisms rather than shell commands (for example, PathExists instead of ls).
+
+**CommandContains**: pass if command output contains string. If it returns an error, check never passes. Use of this check is discouraged.
 
 ```
-type='CommandOutput'
-arg1='(Get-NetFirewallProfile -Name Domain).Enabled'
-arg2='True'
+type = 'CommandContains'
+cmd = 'ufw status'
+value = 'Status: active'
 ```
 
-**CommandContains**: pass if command output contains string. if error, never passes
+**CommandOutput**: pass if command output matches exactly. If it returns an error, check never passes. Use of this check is discouraged.
 
 ```
-type='CommandContains'
-arg1='firewall status'
-arg2='Active'
+type = 'CommandOutput'
+cmd = '(Get-NetFirewallProfile -Name Domain).Enabled'
+value = 'True'
 ```
 
-**PathExists**: pass if specified path exists. This works for both files AND folders (directories).
+**DirContains**: pass if directory contains regex string
```

Comment:
The fact that this check and `FileContains` are both regex is only barely documented, is there any way to make this more  clear?
Regex support is something I use a lot in this SE, and I was initially scared because I didn't see the `*Regex` checks (and I feel like other regex-heavy image creators will miss this too)
Also, can you talk a little bit about what regex standard it uses? Because I still have yet to figure it out, and especially on Windows, there is like 10 different "regex" standards.